docs/janssen-server/install/helm-install/prerequisites/rancher.md

```
@@ -6,66 +6,95 @@ tags:
+    wget https://raw.githubusercontent.com/JanssenProject/jans/vreplace-janssen-version/automation/pgsql.yaml
+    kubectl apply -f pgsql.yaml # adjust values as preferred
+    ```
+
+#### Option B: Install on VM (Docker/No PV Support)
+
+
+Use this option if you are running a single-node Docker test environment or lack PV support. You can install the database package(MySQL/PostgreSQL) directly on your Linux VM.
+
+### 2. Install [Nginx-Ingress](https://github.com/kubernetes/ingress-nginx)
+    
 ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)